content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -14,16 +14,16 @@ weight: 50
 <!-- overview -->
 
 A Job creates one or more Pods and will continue to retry execution of the Pods until a specified number of them successfully terminate.
-As pods successfully complete, the Job tracks the successful completions.  When a specified number
-of successful completions is reached, the task (ie, Job) is complete.  Deleting a Job will clean up
+As pods are successfully completed, the Job tracks the successful completions.  When a specified number
```

Comment:
```suggestion
As pods are successfully completed, the Job tracks the successful completions.  When a specified number
```
The original adheres to the [Kubernetes style guide on simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)